requirements.txt

```diff
@@ -1,16 +1,20 @@
 appdirs==1.4.3
 attrs==17.2.0
+autobahn==17.9.3
 certifi==2017.7.27.1
 chardet==3.0.4
 click==6.7
 HeapDict==1.0.0
 idna==2.6
 python-dateutil==2.6.1
+pytz==2017.2
 requests==2.18.4
 six==1.11.0
+SQLAlchemy==1.1.14
```

Comment:
I'm trying to keep the requirements minimal so I made `SQLAlchemy` an optional extra in the `setup.py` file.

Do you know which packages require `autobahn` and `txaio`?

Reply:
You can ignore autobahn (that was picked up by mistake). Autobahn is the WAMP implementation for Python. WAMP is in turn a weird WS protocol that Poloniex used in their v1 version (which does not work anymore) of their WS feed.

Reply:
However, SQLAchemy is needed. If I don't install it, and try run it inside the container, it complains!

Reply:
Yes, but it's an optional dependency. You install it with:

    pip install numismatic[SQL]

It's already specified in the `setup.py` file in the `extras_require` directive.

I want to change the `SQL` to `sql` actually. There's a visualisation piece I want to add which will require `pandas` and I'll put that change in at the same time.

Reply:
Two things:

1. I am not familiar with optional dependencies, but if it is optional, then it should work without it needing to be installed. This is not the case now. So it's not optional (in my mind)
2. Installing pandas will require the container be much more complex than it already is. I am against the visualization for two reasons: (a) It will complicate the container (b) I think visualization should be an "add-on" module, and not part of numismatic cli

Reply:
It is optional in that the FileCollector works without `sqlalchemy` or Pandas. If you want to use the SqlCollector then you have to install the `sql` dependencies. If you want to use the DataframeCollector you'll have to install `pandas`.

Reply:
With regards to the container, you can have a minimal one to run the collector in the background. You could provide another one with more optional extras installed or leave that for people to do themselves.